app/api/kpi.py

```diff
@@ -91,6 +91,16 @@ def assign_kpi_to_user(user_id, kpi_id):
         })
     return jsonify(ret), 200
 
+@bp.route('/users_on_kpi/<string:kpi_id>', methods=["GET"])
+@token.admin_required
+@jwt_required
+def memeber_kpi(kpi_id):
+    users = mongo.db.users.find({
+        "kpi_id": kpi_id
+    })

```

Comment:
this will again sent full profile details its not needed

Reply:
ok sir